pkg/endpoint/endpoint.go

```diff
@@ -246,8 +246,8 @@ func getKineStorageBackend(ctx context.Context, driver, dsn string, cfg Config)
 		backend, err = pgsql.New(ctx, dsn, cfg.BackendTLSConfig, cfg.ConnectionPoolConfig, cfg.MetricsRegisterer)
 	case MySQLBackend:
 		backend, err = mysql.New(ctx, dsn, cfg.BackendTLSConfig, cfg.ConnectionPoolConfig, cfg.MetricsRegisterer)
-	case JetStreamBackend:
-		backend, err = jetstream.New(ctx, dsn, cfg.BackendTLSConfig)
+	case NATSBackend:
```

Comment:
I think we probably need to continue to support the old `jetstream://` URI scheme, for compatibility purposes.
```suggestion
	case JetStreamBackend, NATSBackend:
```